scala/src/main/scala-2.11/org/kineticcafe/NewSet.scala

```diff
@@ -0,0 +1,4 @@
+package org.kineticcafe
+
+class NewSet() {
```

Comment:
Maybe it was conflicting because of a lowercase `set`? http://www.scala-lang.org/files/archive/nightly/docs/library/index.html#scala.collection.script.Reset